pkg/model/cluster_test.go

```diff
@@ -172,9 +172,9 @@ func testCluster(t *testing.T, describe spec.G, it spec.S) {
 		})
 	})
 
-	describe("BufferStock()", func() {
-		it("returns the configured buffer stock", func() {
-			assert.Equal(t, rawSubject.requestsInBuffer, subject.BufferStock())
+	describe("RoutingStock()", func() {
+		it("returns the configured routing stock", func() {
+			assert.Equal(t, rawSubject.requestsInBuffer, subject.RoutingStock())
```

Comment:
`requestsInRouting`

Reply:
Renamed it in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit